gui/src/hooks/tracking-checklist.ts

```diff
@@ -165,6 +166,9 @@ export function provideTrackingChecklist() {
     res.stepId = step;
     res.ignore = ignore;
     sendRPCPacket(RpcMessage.IgnoreTrackingChecklistStepRequest, res);
+    Sentry.metrics.count('mute_checklist_step', 1, {
+      attributes: { step: TrackingChecklistStepId[step] },
+    });
```

Comment:
isn't this function called for both mute and unmute?
```suggestion
    Sentry.metrics.count(ignore ? 'mute_checklist_step' : 'unmute_checklist_step', 1, {
      attributes: { step: TrackingChecklistStepId[step] },
    });
```